src/main/java/com/google/devtools/build/lib/bazel/repository/RepositoryOptions.java

```diff
@@ -124,6 +124,26 @@ public class RepositoryOptions extends OptionsBase {
       help = "Scale all timeouts related to http downloads by the given factor")
   public double httpTimeoutScaling;
 
+  @Option(
+      name = "http_connector_attempts",
+      defaultValue = "0",
+      documentationCategory = OptionDocumentationCategory.BAZEL_CLIENT_OPTIONS,
+      effectTags = {OptionEffectTag.UNKNOWN},
```

Comment:
I'd follow the example of the `http_timeout_scaling` flag and use `BAZEL_INTERNAL_CONFIGURAITON` here

Reply:
Done